text/0000-promote-aarch64-unknown-linux-gnu-to-tier1.md

```diff
@@ -0,0 +1,188 @@
+- Feature Name: `promote-aarch64-unknown-linux-gnu-to-tier-1`
+- Start Date: 2020-07-17
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+Promote the Arm aarch64-unknown-linux-gnu Rust target to Tier-1.
+
+# Motivation
+[motivation]: #motivation
+
+Arch64-unknown-linux-gnu is [currently a Tier-2 Rust target](https://forge.rust-lang.org/release/platform-support.html#tier-2), in accordance with the target tier policy articulated [here](https://rust-lang.github.io/compiler-team/minutes/design-meeting/2019-09-20-target-tier-policy/).
+
+In the last 2 quarters, very good progress has been made in understanding and filling the gaps that remain in the path to attaining Tier-1 status for this target.
+
+As a direct result, those gaps have either already been filled or are very close to being filled.
+
+As such, this RFC aims to:
+
+- Evidence what has been done.
```

Comment:
_Give evidence of_ what has been done?

Reply:
The use of `evidence` as a transitive verb is considered legal! (see: https://www.merriam-webster.com/dictionary/evidence). A purist would argue that that's not the case with the British English definition though! :)

Reply:
Agh, let them eat US English!

Reply:
I love wacky verbs like this. 💖